src/it/resources/org/checkstyle/suppressionxpathfilter/throwscount/SuppressionXpathRegressionThrowsCount3.java

```diff
@@ -0,0 +1,22 @@
+package org.checkstyle.suppressionxpathfilter.throwscount;
+
+public class SuppressionXpathRegressionThrowsCount3 {
+
+    public void myFunc() throws ArithmeticException,
+            NumberFormatException { // ok
+        // body
+    }
+
+    private void privateFunc() throws CloneNotSupportedException, // warn, max allowed is 4
+            ClassNotFoundException,
+            IllegalAccessException,
+            ArithmeticException,
+            ClassCastException {
+        // body
+    }
+
+    private void func() throws IllegalStateException,
+            NullPointerException { // ok
```

Comment:
Remove all code without warn. We test only xpath, all other code is not required